.github/ISSUE_TEMPLATE/Standard-Bug-Issue-Template.md

```diff
@@ -41,24 +41,21 @@ devchat: < If Hl Engineering requires notification instead of the end user or if
 < list of subject domain experts for this domain. If the subject expert from the end-user perspective is not the author, an end-user expert should be listed here. The Design, Dev, and QA teams should add experts here as appropriate. >
 
 ## Progress
-### Ready for Planning
+### Reviewed Issues - completed by SM & Team Leads
```

Comment:
I think we should be more specific here. Calling this just "Reviewed" could confuse Code Review vs Issue Review.

Reply:
Will change to "Code Reviewed ..."

Reply:
OK, nevermind, I completely spaced on this one. I am changing it to SME Reviewed.